publisher/image/Dockerfile

```diff
@@ -1,8 +1,4 @@
-FROM alpine:3.1
-
-# install curl in the image so it is possible to get the runtime
-# profiling information without any additional package installation.
-RUN apk add --update-cache curl && rm -rf /var/cache/apk/*
```

Comment:
We can't really expose it outside of the container. Some users run deis out on networks that exposes everything to the public (DigitalOcean)